composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
Interesting... but if our test code utilizes phpunit then it's technically a dependency (even if it's just a dev requirement). We maybe need to look at the require-dev part of the composer. This would allow us to run `composer install --no-dev` for releases which will *not* put those dev tools into vendor. But for development I think we'll need a place for tools like phpunit.

Phing seems neat, but I wonder if it's a responsibility the integrators should be tasked with. We can easily install phing globally (which puts it outside the codebase) and then we run it as needed. This way we can keep it out of the codebase. Although I can also argue it too should be listed under the require-dev tag as it is something we need to build